datastore/src/test/java/io/spine/server/storage/datastore/given/DatastoreWrapperTestEnv.java

```diff
@@ -74,10 +59,21 @@ public static void ensureNamespace(String namespaceValue, Datastore datastore) {
         datastore.put(entity);
     }
 
+    /**
+     * Returns the {@code Datastore} instance connected to the Datastore emulator
+     * running {@linkplain TestDatastores#local() locally}.
+     */
     public static Datastore localDatastore() {
         return TestDatastores.local();
     }
 
+    /**
+     * Returns the {@code Datastore} instance connected to the Datastore instance
+     * running remotely.
+     *
+     * <p>The connection to a remote Datastore instance is performed via default service
```

Comment:
As far as I remember "default" service account is the one used on AppEngine (and only there). I suggest just "performed via the service account".